skins/laika/src/scss/overrides.scss

```diff
@@ -1,6 +1,19 @@
 @import "variables";
 
 // Miscellaneous overrides
+.column {
+	padding: 18px;
+}
+
+.navbar-brand {
+	margin-right: 54px;
+}
```

Comment:
I think we discussed this once before and I personally am still not a fan of breaking down the rules like that because they are harder to search for. Plus in this case there are no rules on `navbar` itself so I don't really see the point.

Reply:
I guess my best argument would be "that's how the Bulma files do it". I don't have a super strong opinion on this, I just like it because it's more concise. It's not a blocker.